modules/administration-usage-page-overview.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="administration-usage-page-overview_{context}"]
+= Administration usage page overview
+
+When you click *Show administration usage* in the *System Health* page, you can view the product usage data for the number of secured Kubernetes nodes and CPU units for secured clusters based on metrics collected from Sensors. You can use this information to estimate {rh-rhacs-first} consumption data for reporting.
+
+For more information on how CPU units are defined in Kubernetes, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[CPU resource units] (Kubernetes documentation).
+
+[NOTE]
+====
+{ocp} provides its own usage reports. You can use this information with self-managed Kubernetes systems.
+====
+
+{product-title-short} provides the following usage data in the web portal and API:
+
+Currently secured:: 
+CPU units::: The number of Kubernetes CPU units that your {product-title-short} secured clusters uses, as of the latest metrics collection.
+Node count::: The number of Kubernetes nodes that {product-title-short} secures, as of the latest metrics collection.
+Maximum secured:: 
+CPU units::: The maximum number of CPU units that your {product-title-short} secured clusters use, measured hourly and aggregated for the time period that you specified.
+Node count::: The maximum number of Kubernetes nodes that {product-title-short} secures, measured hourly and aggregated for the time period that you specified.
+CPU units observation date::: The date on which the maximum secured CPU units data was collected.
+Node count observation date::: The date on which the maximum secured node count data was collected.
```

Comment:
I didn't know about this description list variant (`::` and `:::` subordinate). Very cool! :nerd_face: